meshtastic/interfaces/ble/management_runtime.py

```
@@ -0,0 +1,831 @@
+        return _ManagementStartContext(
+            expected_implicit_binding=expected_implicit_binding,
+            target_address=target_address,
+            use_existing_client_without_resolved_address=use_existing_client_without_resolved_address,
+        )
+
+    # COMPAT_STABLE_SHIM: historical helper alias retained for compatibility targets.
+    def _start_management_phase(self, address: str | None) -> _ManagementStartContext:
+        """Compatibility alias for `start_management_phase`."""
+        return self.start_management_phase(address)
+
+    def resolve_management_target(
```

Ruff (0.15.6)

[warning] Too many branches (13 > 12)

(PLR0912)